src/GlobalStyle.tsx

```diff
@@ -3,21 +3,25 @@ import { createGlobalStyle } from "styled-components";
 const GlobalStyle = createGlobalStyle`
 * {
     box-sizing: border-box;
-    margin: 0 auto; 
+    margin: 0; 
```

Comment:
No hagáis esto porque tendréis las devtools llenas de ruido. Hay pocos márgenes que necesitéis resetear, si no queréis hacer un reset de cada cosa podéis cargar un reset o un normalize de terceros.

Reply:
Okei, miraremos de arreglarlo, gracias <3